driver/Cases.jl

```diff
@@ -1129,8 +1152,23 @@ function surface_param_kwargs(::LES_driven_SCM, namelist)
     return (; LESDat)
 end
 
-function ForcingBase(case::LES_driven_SCM, param_set::APS; nudge_tau)
-    ForcingBase(get_forcing_type(case); apply_coriolis = false, coriolis_param = 0.376e-4, nudge_tau = nudge_tau)
+function ForcingBase(
+    case::LES_driven_SCM,
+    param_set::APS;
+    wind_nudge_τᵣ,
+    scalar_nudge_zᵢ,
+    scalar_nudge_zᵣ,
+    scalar_nudge_τᵣ,
+)
+    ForcingBase(
+        get_forcing_type(case);
+        apply_coriolis = false,
+        coriolis_param = 0.376e-4,
+        wind_nudge_τᵣ = wind_nudge_τᵣ,
+        scalar_nudge_zᵢ = scalar_nudge_zᵢ,
+        scalar_nudge_zᵣ = scalar_nudge_zᵣ,
+        scalar_nudge_τᵣ = scalar_nudge_τᵣ,
+    )
 end
```

Comment:
```suggestion
ForcingBase(
    case::LES_driven_SCM,
    param_set::APS;
    kwargs...,
)
    ForcingBase(
        get_forcing_type(case);
        apply_coriolis = false,
        coriolis_param = 0.376e-4,
        kwargs...,
    )
```
Can we get away with this instead?